downstream/modules/navigator/proc-review-ee-tui.adoc

```diff
@@ -0,0 +1,76 @@
+
+[id="proc-review-ee-tui_{context}"]
+
+
+
+= Reviewing {ExecEnvNameStart} from the {Navigator} textual user interface
+
+[role="_abstract"]
+
+You can review your {ExecEnvNameStart} with the {Navigator} textual user interface.
+
+.Prerequisites
+
+* {ExecEnvName}
+
+.Procedure
+
+. Review the {ExecEnvName} included in your {Navigator} configuration.
++
+```
+$ ansible-navigator images
+
+NAME                                                             TAG      EXECUTION ENVIRONMENT       CREATED         SIZE
+0│ansible-automation-platform-20-ee-minimal-rhel8                latest                    True       3 weeks ago     411 MB
+1│ansible-automation-platform-20-ee-supported-rhel8 (primary)    latest                    True       21 hours ago    914 MB
+2│ansible-runner                                                 devel                     True       3 weeks ago     652 MB
```

Comment:
can the runner and network-ee image be removed?  they aren't user facing images

Reply:
maybe include a non-ee container in the list so they are all not true?  

maybe quay.io/ansible/python-base:latest

Reply:
I should also note, navigator will have a default image of it's own in a couple weeks:

https://github.com/ansible/ansible-navigator/issues/141

so we may want to empty docs later (not a critical thing)

Reply:
it's own image?  Not one that is the base image?